Source/Modules/main.cxx

```diff
@@ -826,6 +826,53 @@ static void getoptions(int argc, char *argv[]) {
 
 static void SWIG_exit_handler(int status);
 
+#if defined(_WIN32)
+#  ifndef SWIG_LIB_RELATIVE_PATH
+#    define SWIG_LIB_RELATIVE_PATH "Lib"
+#  endif
+static String *get_swig_lib_path(void) {
+  char buf[MAX_PATH];
+  // TODO: if this is actually in install/bin/, need another call to strrchr
```

Comment:
* [ ] TODO: based on the answer to whether swig.exe is in bin/ when insalled or not, 60f016a34d065aa81034e201a5abf55271c368b4 should be kept (if in bin/) or reverted (if at root)